### ISSUE TITLE
Adjust width of the preview window

### ISSUE THREAD

**Issue by mjlombardi** [nylas/N1#1775]
Apologies if this has already been suggested.. is there a way to adjust the width of the message body? It appears fixed at a max width even if you make the preview pane wider.

**Comment by oojacoboo** [comment]
Def would like to see this.  There isn't enough width at all and limits my efficiency.  Plus is looks weird.  Note that I do have a large monitor.

**Comment by Chyzwar** [comment]
yeah, I have similar problem on my 25 inch dell 2500x1440 it do not scale very well.

**Comment by hallamoore** [comment]
Hmm, actually there are more issues we have to work through with this one, so it'll probably be out with not this upcoming release, but the one after that!
